dts/arm/silabs/efr32fg1p.dtsi

```diff
@@ -17,6 +17,11 @@
 		compatible = "mmio-sram";
 	};
 
+	aliases {
+		usart-0 = &usart0;
```

Comment:
I have one question, maybe related to this. I plan to implement an spi driver that uses the usart peripheral.
In the device tree, this could use the same base node, e.g. usart@40010000, right?

Reply:
Yes, DTS describes underlying hardware and its structure. Since both drivers use the same hardware module they will use the same dts node. 

One example that I know of where the same hardware module is used by several drivers is sercom peripheral on Atmel SAMD21 device. Best place to start looking into the code is [here](https://github.com/zephyrproject-rtos/zephyr/blob/a0b9e70e1095da00f3df69473fca6d0c728431a3/boards/arm/adafruit_feather_m0_basic_proto/adafruit_feather_m0_basic_proto.dts#L36).

Reply:
Thank you